src/lib/y2storage/disk_analyzer.rb

```diff
@@ -374,7 +374,7 @@ def repositories_device_names
     # @param repository [Y2Packager::Repository]
     # @return [Array<String>]
     def repository_device_names(repository)
-      match_data = repository.url.to_s.match(/.*device[s]?=([^&]*)/)
+      match_data = repository.url.to_s.match(/.*devices?=([^&]*)/)
```

Comment:
IMHO, the previous version was more explicit for eyes untrained in the perils of regexps, like mine.

Reply:
well, for me any regexp is write only and only way to verify its behavior is unit testing :)